2-positioning/styles.css

```
@@ -0,0 +1,48 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  position: sticky;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  top: 0px;
```

```suggestion
  top: 0;
```
0 and 0px are the same